modules/dhp_connected_devices/lib/dhp_connected_devices/data_migrations/reset_all_veteran_fitbit_records.rb

```diff
@@ -0,0 +1,14 @@
+# frozen_string_literal: true
+
+module DhpConnectedDevices
+  module DataMigrations
+    module ResetAllVeteranFitbitRecords
+      module_function
+
+      def run
+        # Warning! this will reset ALL veteran records to show they do not have a fitbit connected to their account
```

Comment:
Did we create a card to make this only run in lower environments?

Reply:
see my other comment